src/custom/components/WalletModal/WalletModalMod.tsx

```diff
@@ -283,16 +287,46 @@ export default function WalletModal({
                 icon={MetamaskIcon}
               />
             )
+          } else if (option.name === 'Tally Ho' && tallyInstalled && !tallyIsDefault) {
+            return (
+              <Option
+                id={`connect-${key}`}
+                key={key}
+                color={'#D59B4B'}
+                header={<Trans>Tally Ho</Trans>}
+                subheader={<Trans>To use Tally Ho, enable as default in Tally Ho settings and refresh the page.</Trans>}
+                link={'https://tally.cash/'}
```

Comment:
Could this link to a help article where it instructs the user how to turn this setting on?

Reply:
sounds good